tests/conftest.py

```diff
@@ -251,3 +256,82 @@ def fixture_tower_task() -> TowerTask:
         file_format=FileFormat.JSON, file_path=TowerTaskResponseFile.RUNNING
     )
     return TowerTask(**tower_task_running_content["tasks"][0]["task"])
+
+
+@pytest.fixture(scope="session")
+def slurm_squeue_output() -> Dict[str, Path]:
+    """Return SLURM squeue output for analysis started via SLURM."""
+    return {
+        "blazinginsect": Path(
+            "tests",
+            "fixtures",
+            "squeue",
```

Comment:
can we have a `fixture_directory` so we don't have to repeat this so many times?

Reply:
Good suggestion - fixed